dali/server/daldap.cpp

```diff
@@ -139,7 +139,7 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
         {
             username.append(filesdefaultuser);
             decrypt(password, filesdefaultpassword);
-            OWARNLOG("Missing credentials, injecting deprecated filesdefaultuser");
+            OWARNLOG("Missing credentials, injecting deprecated filesdefaultuser for request %s %s", key ? key : "NULL", obj ? obj : "NULL");
```

Comment:
minor: jstring's nullText is handy/preferable and standarizes the output for situations like this.